include/nvexec/stream/common.cuh

```diff
@@ -496,7 +498,7 @@ namespace nvexec {
               : operation_state_base_t<OuterReceiverId>((outer_receiver_t&&)out_receiver, context_state, true)
               , storage_(queue::make_host<variant_t>(this->status_, context_state.pinned_resource_))
               , task_(queue::make_host<task_t>(this->status_, context_state.pinned_resource_, receiver_provider(*this), storage_.get(), this->get_stream(), context_state.pinned_resource_).release())
-              , started_(ATOMIC_FLAG_INIT)
+              , started_ ATOMIC_FLAG_INIT
```

Comment:
Is this even well-formed?! From cppreference.com, it appears that this flag isn't needed anymore since C++20. https://en.cppreference.com/w/cpp/atomic/ATOMIC_FLAG_INIT

And from [atomic.flags]:

> The macro can be used in the form:
> ```
> atomic_flag guard = ATOMIC_FLAG_INIT;
> ```
> It is unspecified whether the macro can be used in other initialization contexts.

Reply:
This is defined as `#define ATOMIC_FLAG_INIT {false}` in `cuda/std/detail/libcxx/include/atomic`. Wrapping `{false}` in parens triggers a clang warning.

Reply:
Yes, but that syntax is not portable, and cppreference says we don't even need it because default initialization should have the same effect.